main/res/values/strings.xml

```diff
@@ -1527,6 +1527,7 @@
 
     <!-- text-to-speech for compass view -->
     <string name="tts_service">Talking compass</string>
+    <string name="tts_toggle">Toggle talking</string>
```

Comment:
Can't we reuse the `tts_start` and `tts_stop` at this point. If you look into the compass it uses these two strings (depending on current status).

Reply:
Or use "toggle" for compass as well - forgot to change it there.
I don't see a big additional benefit by providing two toggling menu entries (as there is a toast shown which informs the user whether tts started or ended), and it inflates the code unnecessarily IMHO.

Reply:
Agreed. You might change it in the compass as well.
The only advantage I see in using the current strings is, that we do not need to wait for translations...but thats minor IMHO.